api/envoy/config/api_listener/v2/api_listener.proto

```diff
@@ -0,0 +1,24 @@
+syntax = "proto3";
+
+package envoy.config.api_listener.v2;
```

Comment:
I would suggest making this `envoy.config.listener.v2` as a package name. The proto can still be `api_listener.proto`, but this can be a package where general listener config goes in the future.

Reply:
Done.